### PR TITLE
Trend laser calibration from QCG

### PR DESCRIPTION
**Pull request by cimordas**
This draft depends on this [PR](https://github.com/AliceO2Group/AliceO2/pull/7394) to be finalized and tested with clang

### CODE REVIEW

File: Modules/Common/include/Common/TH1Reductor.h [hunk 1]
```diff
@@ -24,7 +24,7 @@ namespace o2::quality_control_modules::common
 /// \brief A Reductor which obtains the most popular characteristics of TH1.
 ///
 /// A Reductor which obtains the most popular characteristics of TH1.
-/// It produces a branch in the format: "mean/D:stddev:entries"
+/// It produces a branchin the format: "mean/D:stddev:entries"
```
Comment (wiechula) [inline Modules/Common/include/Common/TH1Reductor.h:27]: ?

File: Modules/TPC/run/tpcQCTrending_laserCalib.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
```
Comment (Barthelemy) [inline Modules/TPC/run/tpcQCTrending_laserCalib.json:19]: We used to have this url everywhere but it spams the DNS for no useful reason. Could you just use an empty url ? 
Thank you

Comment (cimordas) [reply Modules/TPC/run/tpcQCTrending_laserCalib.json]: I changed the url to an empty string `""`. Is this right?

**Review by wiechula — APPROVED** [review]
A bit ugly, but ok for the moment I think. Actually, it would be nice if the framework would allow for using any type in the trending tree which has a ROOT dictionary, not only basic types.

File: Modules/Common/include/Common/TH1Reductor.h (hunk 1 above)
Comment (cimordas) [reply Modules/Common/include/Common/TH1Reductor.h]: I think this fix got reverted when I updated my QC yesterday, as I remember fixing it already once. Now it should be fine

File: Modules/TPC/run/tpcQCTrending_laserCalib.json (hunk 2 above)
Comment (Barthelemy) [reply Modules/TPC/run/tpcQCTrending_laserCalib.json]: yes that's good

**Comment by Barthelemy** [comment]
@wiechula Do not hesitate to create a ticket in JIRA, we consider all requests !

**Pull request merged**